frontend/src/components/stat/chart/LineChart.tsx

```diff
@@ -1,25 +1,25 @@
 import { ChartData, ChartOptions } from "chart.js";
 import _ from "lodash";
 import { Line } from "react-chartjs-2";
-import { DataLabelsOptions } from "./DataLabelsOptions";
+import { DataLabelsOptionsTop } from "./DataLabelsOptions";
+import { DefaultChartOptions } from "./options/DefaultChartOptions";
 
 export default function LineChart(props: {
     data: ChartData<"line", any[], string>;
     options: ChartOptions<"line">;
 }) {
-    const defaultOptions: ChartOptions<"line"> = {
-        layout: {
-            padding: 25,
-        },
-        plugins: {
-            datalabels: DataLabelsOptions,
-            legend: { position: "bottom" },
-        },
-    };
+    const chartOptions: ChartOptions<"line"> = _.merge({}, 
+        DefaultChartOptions, {
+            plugins: {
+                datalabels: DataLabelsOptionsTop
+            }
+        }
+    )
+
     return (
         <Line
             data={props.data}
-            options={_.merge(defaultOptions, props.options)}
+            options={_.merge({}, chartOptions, props.options)}
```

Comment:
the `_.merge` enjoyer has LOGGED in!

Reply:
btw, another stupid moment, but if you don't pass `{}` in the first argument like I did here, it mutates `defaultOptions` as well as returning a new object. Where is my functional programming lodash function.